lib/control/v1/health.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const HTTP_OK = 200;
+const HTTP_METHOD_NOT_ALLOWED = 405;
+const started = Date.now();
+
+function Health(app) {
+  const routes = {
+    health: app.route('/v1/health')
+  };
+  const allowedMethods = 'GET';
+
+  /**
+   * Sets headers and status for routes that should return a 405
+   * @param {Express.req} req
+   * @param {Express.res} res
+   */
+  const methodNotAllowed = (req, res) => {
+    res.set('Allow', allowedMethods);
+    res.status(HTTP_METHOD_NOT_ALLOWED);
+    res.end();
+  };
+
+  routes.health.get((req, res) => {
+
+    res.status(HTTP_OK).json({
+
+      status: 'okay',
```

Comment:
The only thing I can think of would be if Warden couldn't contact the vault cluster. but seeing as it is connecting to local host, do we need another check for that?